### PR TITLE
Add ics

### PR DESCRIPTION
**Pull request by ickc**
@conda-forge/help-python

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/ics/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (ocefpaf) [inline recipes/ics/meta.yaml:1]: Please remove the comments.

File: recipes/ics/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3.6
+    - 'arrow <0.15,>=0.11'
```
Comment (ocefpaf) [inline recipes/ics/meta.yaml:25]: You can need the quotes.

Comment (ocefpaf) [reply recipes/ics/meta.yaml:25]: Also, you probably don't need this whole build section, only `host`.

**Comment by ickc** [comment]
@ocefpaf, thanks. Ready to merge.

File: recipes/ics/meta.yaml (hunk 2 above)
Comment (ocefpaf) [reply recipes/ics/meta.yaml]: Did you try removing the build section here? I'm pretty sure you don't need it. Only host would suffice.

Comment (ickc) [reply recipes/ics/meta.yaml]: Yes, I removed them in the latest commits. The one you're referring to is outdated.

Comment (ocefpaf) [reply recipes/ics/meta.yaml]: Nope. They are still there. I removed them in https://github.com/conda-forge/staged-recipes/pull/13465/commits/4e1181b75a13d86fef68ba6c677bf1ece424721e

**Pull request merged**